client/src/actions/geneset.ts

```diff
@@ -54,16 +55,9 @@ export const genesetAddGenes =
   (genesetName: any, genes: any) => async (dispatch: any, getState: any) => {
     const state = getState();
     const { obsCrossfilter: prevObsCrossfilter, annoMatrix } = state;
-    const { schema } = annoMatrix;
-    const varIndex = schema.annotations.var.index;
-    const varLabel = "feature_name";
-    const varColumns = annoMatrix.getMatrixColumns("var");
-
-    const labelToUse = varColumns.includes(varLabel) ? varLabel : varIndex;
-    const dfNames: Dataframe = await annoMatrix.fetch("var", labelToUse);
-
-    const geneNames = dfNames.col(labelToUse).asArray();
-
+    const varFeatureName = VAR_FEATURE_NAME_COLUMN;
```

Comment:
sanity check - maybe we could just use `VAR_FEATURE_NAME_COLUMN` directly at callsite?

```ts
const geneNames = df.col(VAR_FEATURE_NAME_COLUMN).asArray();
```